Formula/n/nzbget.rb

```diff
@@ -20,58 +20,39 @@ class Nzbget < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "65fd679c775ee8595c57e5459cc0eba8802944aa7181a4548e6cc6ad0340ffb3"
   end
 
-  disable! date: "2023-10-18", because: :repo_archived
-
-  depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
+  depends_on "boost"
   depends_on "openssl@3"
+  depends_on "sevenzip"
 
   uses_from_macos "libxml2"
   uses_from_macos "ncurses"
 
-  # Fix OpenSSL 3 compatibility
-  # upstream PR ref, https://github.com/nzbget/nzbget/pull/793
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/56a864d/nzbget/openssl-3.patch"
-    sha256 "7fd5e300c6ba456df20307a2d3de630e3cb6d5dfdc2662abd567190eb55ac3be"
-  end
-
   def install
-    ENV.cxx11
-
-    # Fix "ncurses library not found"
-    # Reported 14 Aug 2016: https://github.com/nzbget/nzbget/issues/264
-    if OS.mac?
-      (buildpath/"brew_include").install_symlink MacOS.sdk_path/"usr/include/ncurses.h"
-      ENV["ncurses_CFLAGS"] = "-I#{buildpath}/brew_include"
-      ENV["ncurses_LIBS"] = "-L/usr/lib -lncurses"
-    else
-      ENV["ncurses_CFLAGS"] = "-I#{Formula["ncurses"].opt_include}"
-      ENV["ncurses_LIBS"] = "-L#{Formula["ncurses"].opt_lib} -lncurses"
-    end
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "strip", "build/nzbget"
```

Comment:
can you move this to your cmake build config?

Reply:
We have to make neccessary changes to nzbget first to comply, we'll be happy to do that with v25 (next release). v24 has just been released and can't be easily updated.

Reply:
Any issue for tracking this, can you add that as code comments? Thanks!

Reply:
Tracking issue https://github.com/nzbgetcom/nzbget/issues/257 - comments added

Reply:
thanks a lot!